gnosis/safe-react
# Fix Modules link

## What it solves
Resolves #2875

## How this PR fixes it
The [old Modules link](https://docs.gnosis.io/safe/docs/contracts_architecture/#3-module-management) was updated to the [new one](https://docs.gnosis-safe.io/contracts/modules-1) and changed to a constant, outside of the component.

## How to test it
Open the Settings panel, then Advanced and click "Learn about more about modules **HERE**". Our new GitBook docs should open with readable content, NOT a 404 page.
